TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/ScoringAlgorithm.java

```diff
@@ -17,8 +17,12 @@ public class ScoringAlgorithm extends OpMode {
     boolean pixelsInClaw = false;
     double armStartPos = 0.0;
     double armTargetPos = 0.0;
+    double armStartTolerance = 0.0;
+    double armTolerance = 0.0;
     double viperSlideStartPos = 0.0;
     double viperSlideTargetPos = 0.0;
+    double viperSlideStartTolerance = 0.0;
+    double viperSlideTolerance = 0;
```

Comment:
this won't compile, you're trying to parse an integer as a double

Reply:
Oopys